repos/system_upgrade/common/actors/redhatsignedrpmscanner/actor.py

```diff
@@ -3,25 +3,39 @@
 from leapp.models import InstalledRedHatSignedRPM, InstalledRPM, InstalledUnsignedRPM
 from leapp.tags import FactsPhaseTag, IPUWorkflowTag
 
-
-class RedHatSignedRpmScanner(Actor):
-    """Provide data about installed RPM Packages signed by Red Hat.
+VENDOR_SIGS = {
+    'rhel': ['199e2f91fd431d51',
+             '5326810137017186',
+             '938a80caf21541eb',
+             'fd372689897da07a',
+             '45689c882fa658e0'],
+    'centos': ['24c6a8a7f4a80eb5',
+               '05b555b38483c65d',
+               '4eb84e71f2ee9d55'],
+}
+
+VENDOR_PACKAGERS = {
+    'rhel': 'Red Hat, Inc.',
+    'centos': 'CentOS',
+}
+
+
+class VendorSignedRpmScanner(Actor):
+    """Provide data about installed RPM Packages signed by the vendor.
 
     After filtering the list of installed RPM packages by signature, a message
     with relevant data will be produced.
     """
 
-    name = 'red_hat_signed_rpm_scanner'
+    name = 'vendor_signed_rpm_scanner'
     consumes = (InstalledRPM,)
     produces = (InstalledRedHatSignedRPM, InstalledUnsignedRPM,)
```

Comment:
I didn't change those (yet), as this would make the diff huge, for no good reason.